src/lib.rs

```diff
@@ -112,16 +112,20 @@ pub const FEATURES_VERSION: u16 = 1;
 #[derive(Error, Clone, Debug, PartialEq, Eq)]
 /// Umbrella type for bliss error types
 pub enum BlissError {
-    #[error("error happened while decoding file – {0}")]
+    #[error("error happened while decoding file - {0}")]
     /// An error happened while decoding an (audio) file.
     DecodingError(String),
-    #[error("error happened while analyzing file – {0}")]
+    #[error("error happened while analyzing file - {0}")]
     /// An error happened during the analysis of the song's samples by bliss.
     AnalysisError(String),
     #[error("error happened with the music library provider - {0}")]
     /// An error happened with the music library provider.
     /// Useful to report errors when you implement bliss for an audio player.
     ProviderError(String),
+    // #[cfg(feature = "symphonia")]
+    #[error("error happened while decoding file - {0}")]
+    /// An error happened while decoding an (audio) file.
+    SymphoniaDecoderError(String),
```

Comment:
Do we really need this you think? Wouldn't the `DecodingError` be enough (especially since in the {0} there will probably be some symphonia-specific information anyways)

Reply:
That makes sense, probably better to minimize changes to the public API

Reply:
Okay looked again, I think I just forgot to remove that.
The conversion from SymphoniaDecoderError to BlissError already uses the BlissError::Decoder variant